test_module/simple_test.py

```diff
@@ -0,0 +1,16 @@
+"""
+Module simple_test
+================
```

Comment:
Add equal signs to match title length

```suggestion
==================
```